### ISSUE TITLE
Question about logits 

### ISSUE THREAD

**Issue by jisu0123** [markus93/NN_calibration#4]
Hello.
First of all, thank you for releasing a really good code. 
I tried to your code using logit files you uploaded.
However, the logit files can be downloaded via 'git-lfs' because its size is so large.
Without purchasing, I cannot download your files..

Sorry for bothering you, but could you upload the logit files to somewhere else ..?
I know this is so time-consuming..

Thank you!
